src/views/Profile/components/EditProfileModal/ChangeProfilePicPage.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useState } from 'react'
+import { Button, InjectedModalProps, Skeleton, Text } from '@pancakeswap-libs/uikit'
+import { useWallet } from '@binance-chain/bsc-use-wallet'
+import { useDispatch } from 'react-redux'
+import nftList from 'config/constants/nfts'
+import { useProfile, useToast } from 'state/hooks'
+import useI18n from 'hooks/useI18n'
+import { fetchProfile } from 'state/profile'
+import useGetWalletNfts from 'hooks/useGetWalletNfts'
+import useApproveConfirmTransaction from 'hooks/useApproveConfirmTransaction'
+import { usePancakeRabbits, useProfile as useProfileContract } from 'hooks/useContract'
+import { getPancakeProfileAddress, getPancakeRabbitsAddress } from 'utils/addressHelpers'
+import SelectionCard from '../SelectionCard'
+import ApproveConfirmButtons from '../ApproveConfirmButtons'
+
+type ChangeProfilePicPageProps = InjectedModalProps
```

Comment:
interface is not working for this case ?

Reply:
I just don't like `interface ChangeProfilePicPageProps extends InjectedModalProps {}`

Throws a eslint warning.